dbms/src/Storages/DeltaMerge/tests/MultiSegmentTestUtil.h

```diff
@@ -106,21 +106,21 @@ class MultiSegmentTestUtil : private boost::noncopyable
         }
         {
             // Check there is only one segment
-            ASSERT_EQ(store->segments.size(), 1);
+            ASSERT_EQ(store->segments.size(), static_cast<unsigned long>(1));
```

Comment:
try remove the `#include <gtest/gtest.h>` line (which is already included in `TiFlashTestBasic.h` with ignoring the sign-compare), and you can get rid of writing these casts